centreon/www/api/class/centreon_configuration_contactgroup.class.php

```diff
@@ -99,6 +99,10 @@ public function getList()
 
         $contactgroupList = array();
         foreach ($aclCgs['items'] as $id => $contactgroup) {
+            if (($this->arguments['type'] == 'local') && ($contactgroup['cg_type'] == 'ldap')) {
```

Comment:
Add a comment to explain please.
```suggestion
            if (($this->arguments['type'] === 'local') && ($contactgroup['cg_type'] === 'ldap')) {
```